e2e/nextjs/app-router/app/app/normal/devcycle.ts

```diff
@@ -20,3 +22,8 @@ export const {
     },
     options: { enableStreaming: false },
 })
+
+export const getFlag = setupDevCycleVercelFlagHelper({
```

Comment:
Why as a separate import and not a part of `setupDevCycle()`?

Reply:
because it depends on the `@vercel/flags` package which I wanted to make an optional dependency. So the idea is that if you want to use it with Vercel, you install their flags package and import this helper from `@devcycle/nextjs-sdk/vercel` where it expects the package to be available.